values.yaml

```
@@ -465,6 +488,24 @@ connectInject:
   #     namespace-label: label-value
   namespaceSelector: null
 
+  # allowNamespacesList is a list of k8s namespaces to allow Connect sidecar
+  # injection in. If a k8s namespace is not included or is listed in `denyNamespacesList`,
+  # pods in that k8s namespace will not be injected even if they are explicitly
+  # annotated. Use "*" to automatically allow all k8s namespaces.
```

```suggestion
  # annotated. Use ["*"] to automatically allow all k8s namespaces.
```

Might be a good idea to show them it needs to be in an array so they don't set it to the string

---

Updated.